docs/tutorial/index.md

```diff
@@ -1,74 +1,48 @@
+With Ubuntu Pro for WSL (UP4W) an [Ubuntu Pro](https://ubuntu.com/pro) subscription empowers you to manage Ubuntu WSL instances at scale.
 
-<br/>
-
-**What you'll need:**
-
-- A Windows 11 machine with access to the Microsoft Store and a minimum of 16GB of RAM and 8-core processor.
-
-- Familiarity with the Windows PowerShell.
-
-<br/>
-
-**What you'll do:**
-
-- Set things up
-
-- Watch UP4W transform your Ubuntu Pro game on WSL!
-
-- Tear things down
-
-<br/>
+In this tutorial you will develop an understanding of how UP4W can be installed and deployed for managing multiple WSL instances.
+To complete this tutorial you will need a Windows 11 machine with a minimum of 16GB RAM and a 8-core processor.
 
 ```{note}
-Throughout this tutorial we'll present shell commands together with their output:
-
-- The commands will be prefixed by a
-prompt matching the shell it's launched in. 
-
-  For example, `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`, thus running on Windows.
+WSL enables using a Linux shell and Windows PowerShell side-by-side on the same machine.
+Throughout this tutorial, commands will be prefixed by a prompt that indicates the shell being used, for example:
 
+- `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`.
 
-  Similarly, a Linux shell prompt logged in as root with the current working directory as the root home directory will look like `root@hostname:~#`
+- `root@hostname:~#` indicates a Linux shell prompt with login as root in the root home directory `~`.
 
-- The outputs are preserved to give you more context.
+Output logs are included in this tutorial when instructive but are typically omitted to save space.
 ```
 
-
 ## Set things up
 
-### Install the `Windows Subsystem for Linux` (WSL) application
-
-```{warning}
-**If you already have it pre-installed:** 
-
-Please reset its configuration to the default settings by (making a backup of your existing `~\.wslconfig` file, then) running:
+### Install WSL and Ubuntu
 
+WSL can be installed directly from the [Microsoft Store](https://www.microsoft.com/store/productId/9P9TQF7MRM4R).
 
-    PS C:\Users\me\tutorial> Remove-Item ~\.wslconfig
+If you already have WSL installed you are advised to backup `~\.wslconfig` and remove it before continuing the tutorial:
 
+```text
+PS C:\Users\me\tutorial> Remove-Item ~\.wslconfig
 ```
 
-
-See: [Microsoft Store | Windows Subsystem for Linux](https://www.microsoft.com/store/productId/9P9TQF7MRM4R)
-
-All set. From now on you can use it to launch  WSL instances.
-
-<br />
-
-### Install the `Ubuntu 22.04 LTS` and `Ubuntu (Preview)` applications
+Ubuntu can also be installed from the Microsoft Store.
+Choose [Ubuntu 22.04 LTS](https://www.microsoft.com/store/productId/9PN20MSR04DW) or [Ubuntu (Preview)](https://www.microsoft.com/store/productId/9P7BDVKVNXZ6)
```

Comment:
@jibel , I was going to look into updating aspects like Ubuntu versions in a later PR but can do this here also if it's preferable.

Question: does this tutorial require both an LTS and a Preview version to be used? It would be more simple to use 24.04 alone if possible.

Reply:
No, this tutorial only requires 24.04 (until the solution is available on all the releases)

Reply:
Well, there is landscape in this tutorial which makes the whole thing confusing. That's why Landscape would be better elsewhere.

Reply:
I think we can settle to move the Landscape setup out of this in a subsequent pull request.